### ISSUE TITLE
Feature request: save images locally

### ISSUE THREAD

**Issue by nathonius** [inhumantsar/slurp#45]
Currently the plugin links to the remote images from where they were slurped. It would be great to have an option to save the images alongside the markdown for eventual case where the url changes or the content is removed.

**Comment by inhumantsar** [comment]
thanks! I like this idea. any thoughts on how images should be organized? I normally keep images for obsidian md files in a subdir called `_files`, not sure how others handle them though

**Comment by Truncated** [comment]
I use Local Images Plus to automatically process images from slurped content. It works a treat on Slurped results.
https://github.com/Sergei-Korneev/obsidian-local-images-plus

As of this writing...
It addresses:
* Leaving the original link name in the content, while changing the file name for your records.
* Option to generate file names as MD5 Hash to avoid name collisions
* Can convert PNG to JPEG if that's your jam
* Manage Media folder target and naming with some amount of variables.

It does not address :
* No deduplication options if you're not using the MD5 + same save location.
* Huge files will choke it, but that feels like a fringe case for most.
* No specific conversions for WEBP.

**Comment by inhumantsar** [comment]
ooo, i'll look into adding an option to call that automatically after writing the file. that would definitely simplify things.

**Comment by nathonius** [comment]
Yeah, adding in a post-process hook to do that (or any other command 👀) would be fantastic

**Comment by Rai2en** [comment]
You can use the amazing plugin "locale image plus". It will do it automatically, and for all files in your vault if you want. Thank me later ;)
